Week_01_Basics/homework-portfolio/portfolio.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<head>
+	<link href="./style.css" rel="stylesheet">
+	<title>Portfolio | Karmaela by Michelle Zhang</title>
+	<link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>
+</head>
+<body>
+	<a name="back">
+	<a href="http://www.karmaela.com" target="_blank"><img id="logo" src="./header-logo.png" alt="Karmaela.com"></a>
+	
+	<nav>
+		<a href="./about.html">About</a> | 
+		<a href="./portfolio.html">Portfolio</a> | 
```

Comment:
You can remove the href attribute here since it's really just linking to its own page.